dev/git_maintenance_tasks

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+set -e
+
+BRANCH=master
+BRANCH=maintenance-test
+
+TOPDIR="$(readlink -e "$(dirname $0)/..")"
+
+# Force C locale
+export LC_ALL=C
+
+mkdir -p ~/lmms-repo
+cd ~/lmms-repo
+
+if [ -e lmms ]
+then
+	cd lmms
+	PULLRES=$(git pull)
```

Comment:
@jasp00 should we consider appending `git checkout $BRANCH` to force us back onto `master` once ready?